README.md

```diff
@@ -255,6 +255,11 @@ Split your app module into different small modules, and give those modules as de
 
 <br/>Read more [here](https://medium.com/swlh/modularization-by-feature-and-layer-with-android-architecture-components-80bf317d737).
 
+28) Avoid using too many Base classes. Using base classes everywhere makes a tight web in your codebase, which later makes it hard to do changes in the code. If still necessary, create and use a standalone function (kotline file). 
```

Comment:
Hey this example looks perfect. A few spelling mistakes. Could you pleas correct it? I wll merge then. Thank you once again :)

Reply:
Did some changes, please have a look and let me know.

Reply:
Thank you for the contribution. Have merged it :)